.github/workflows/ci.yml

```diff
@@ -239,22 +239,22 @@ jobs:
             target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
       - name: Install Forc
-        run: cargo install --locked --debug --path ./forc
+        run: cargo install --locked --path ./forc
       - name: Install Forc plugins
```

Comment:
nit: coulduse a more programmatic approach to get the plugins; i.e. loop through all dir names in ./forc-plugins

Reply:
It makes sense to me, but I am thinking, this PR is already doing two things, and also, there might be a reason to have the plugins explicitly listed. So, I've rather created a separate issue for this change: #7169.